tests/unit/CRR/crrExistingObjects.js

```diff
@@ -75,6 +75,7 @@ describe('crrExistingObjects', () => {
         process.env.KEY_MARKER = 'testKeyMarker';
         process.env.VERSION_ID_MARKER = 'testVersionIdMarker';
         process.env.DEBUG = '0';
+        process.env.CURRENT_VERSION_ONLY = 'true';
```

Comment:
this changes the behavior for CRR as well, which will affect S3C unexpectedly (and will be an issue when CRR is used in Zenko as well)

Can we make this behavior apply only for non-CRR replication(s) ?